### PR TITLE
feat: Updated based on ECS workshop experience

### PR DESCRIPTION
**Pull request by Hoseong-Seo**
*Issue #, if available:*

*Description of changes:*
- SBT and ECS are installed completely independently
- README updated - High level architecture update and so on. (Korean version added)
- Provisioning Source uploaded script added
- Advanced tier ECS is provisioned at initial installation. 
- tenant id pattern changed: tenant name + id
- Github'Security applied 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: images/archi-high-level.png
Comment (dhammikasriya-aws) [inline images/archi-high-level.png:1]: Great effort on this architecture diagram. Given this is our main Diagram that explains ECS SaaS its great other improve that in a way it make sense. 

1. Good to have CoreAppPlane to go with SBT naming convention. Pls consider renaming : 
SaaS Control plane (SBT) 
SaaS Application plan 
- Core Application Plane (SBT) 
- ECS Application plane 
--- Basic 
--- Advanced 
--- Premium 
2. Remove Margate images from Advanced and Premium, and a caption "Amazon ECS" would be better next to to the icon
3. Remove S3 bucket and Rout53 icons, as they might not add value as in this level diagram 
4. Remove "Tenant" that is in between Cognito and Admin console as it is misleading both tenants  and SaaS provider connects to the same Admin console. And, rename Admin console into SaaS Admin console. If we want to showcase Tenant admin panel, we can consider add it next to the "Tenant" that is connecting to the SaaS app .

**Review by dhammikasriya-aws — COMMENTED** [review]
Pls review, one comment about the archi. diagram.

File: images/archi-high-level.png
Comment (Hoseong-Seo) [reply images/archi-high-level.png:1]: Yes, I'll make it more simple at next PR as we discussted.

**Review by dhammikasriya-aws — APPROVED** [review]
Main architecture diagram changes will be handled separately. Rest is Good!

**Pull request merged**